README.md

```diff
@@ -55,7 +55,52 @@ var moddleDescriptor = {
 
 registerBpmnJSModdleExtension(moddleDescriptor);
 ```
+> [!NOTE]
+> This feature requires **Camunda Modeler 5.29+**.
+
+## UI Components
+These components are available for use:
+
+- **Fill**
+- **Modal**
+- **Overlay** 
+- **Section**
+- **ToggleSwitch**
+- **TextInput**
+- **CachedComponent**
+
+Components exported for use in Camunda Modeler client plugins.
+
+```javascript
+import Fill from 'camunda-modeler-plugin-helpers/components/Fill.js';
```

Comment:
This relies on #31, which we have not merged (yet). I'd consider to not document this (at this stage).

Reply:
Alternatively consider this PR blocked until we verify that #31 works and is merged.

Reply:
I'll convert this PR to draft since #31 is a draft.